sdk/cosmos/azure-cosmos/azure/cosmos/aio/_auth_policy_async.py

```diff
@@ -18,6 +18,16 @@
 
 
 class AsyncCosmosBearerTokenCredentialPolicy(AsyncBearerTokenCredentialPolicy):
+    AadDefaultScope = "https://cosmos.azure.com/.default"
```

Comment:
Don't dup the const.

Reply:
We already have a `constants.py` file - seems like a better home for it.